sklearn/semi_supervised/label_propagation.py

```diff
@@ -358,6 +362,18 @@ class distributions will exceed 1 (normalization may be desired).
             affinity_matrix /= normalizer[:, np.newaxis]
         return affinity_matrix
 
+    def fit(self, X, y):
+        # alpha is deprecated for LabelPropagation because it doesn't have any
+        # theoretical meaning (from the reference paper). Look at PR 6727.
+        if self.alpha is not None:
+            warnings.warn(
+                "Deprecated in 0.19 and it's going to be removed in 0.21.",
+                DeprecationWarning
+            )
```

Comment:
In the case of `LabelPropagation`, setting a different `alpha` value is not something supported by the algorithm's description in the cited references.